OMEdit/OMEditLIB/FMI/FMUExportOutputWidget.cpp

```diff
@@ -504,14 +506,18 @@ void FmuExportOutputWidget::zipFMU()
 
   QString program = "zip";
   QStringList arguments;
-  arguments  << "-r" << mFmuLocationPath << ".";
+  arguments << "-r" << mFmuLocationPath << ".";
 
   // check for fmiSources=false and do not export the source folder
   if (!OptionsDialog::instance()->getFMIPage()->getIncludeSourceCodeCheckBox()->isChecked()) {
-    arguments << arguments << "-x" << "'sources/*'";
+#ifdef Q_OS_WIN
+    arguments << "-x" << "'sources/*'";
+#else
+    arguments << "-x" << "sources/*";
+#endif
   } else {
     // ignore the build_cmake directory
-    arguments << arguments << "-x" << "'sources/build_cmake/*'";
+    arguments << "-x" << "'sources/build_cmake/*'";
```

Comment:
@adeas31 we should also do the same thing for excluding the `build_cmake` directory otherwise it will not be excluded in linux, also the arguments must be appended

#ifdef Q_OS_WIN
    arguments << arguments << "-x" << "'sources/*'";
#else
    arguments <<  arguments <<"-x" << "sources/*";
#endif
} else {
#ifdef Q_OS_WIN
    arguments << arguments << "-x" << "'sources/build_cmake/*'";
#else
    arguments <<  arguments <<"-x" << "sources/build_cmake/*";
#endif
}

Reply:
Why do we need to append the arguments?

Reply:
Sorry that was a mistake now I understood that I am duplicating the arguments , but other than that it looks fine